src/plugins/analysis/cve_lookup/code/cve_lookup.py

```diff
@@ -230,11 +232,32 @@ def versions_match(cpe_version: str, cve_entry: CveDbEntry) -> bool:
     return True
 
 
+def coerce_version(version: str) -> Version:
+    '''
+    The version may not be PEP 440 compliant -> try to convert it to something that we can use for comparison
+    '''
+    try:
+        return parse_version(version)
+    except InvalidVersion:
+        # try to convert other conventions (e.g. debian policy) to PEP 440
+        fixed_version = version.lower().replace('~', '-').replace(':', '!', 1).replace('_', '-')
+    try:
+        return parse_version(fixed_version)
+    except InvalidVersion:
+        match = VALID_VERSION_REGEX.match(fixed_version)
+        if match:
+            valid_version = match.group()
+            rest = re.sub('[^a-zA-Z0-9._-]', '', fixed_version[len(valid_version) :]).lstrip('._-')
```

Comment:
`\w=[A-Za-z0-9_]` --> Maybe a little simpler here

```suggestion
            rest = re.sub('[^\w.-]', '', fixed_version[len(valid_version) :]).lstrip('._-')
```

Reply:
fixed (with a raw string though see [re](https://docs.python.org/3/library/re.html))